packages/cli/package.json

```diff
@@ -94,6 +91,7 @@
     "@types/prompts": "^1.1.1",
     "jest": "^29.7.0",
     "oclif": "^4.17.32",
+    "semver": "^7.7.2",
```

Comment:
used in a script

Reply:
the homebrew script

Reply:
can we get rid of it or make the script use NPX maybe?

Reply:
that is so in the weeds i think we should not spend time doing that

Reply:
its serving a good purpose.

Im focusing on being glad we found this before we went to release and home brew script failed.